### ISSUE TITLE
Drag ruler not showing color coding with waypoints.

### ISSUE THREAD

**Issue by Lockbox313** [manuelVo/foundryvtt-drag-ruler#308]
When using the  Drag Ruler module  with RuneQuest system the color coding from movement green/yellow/red is not working when waypoints are used.  Does this have something to with Drag Ruler's API no longer supporting the function getCostForStep as noted in the  release notes.

**Comment by manuelVo** [comment]
That function is only relevant when both Enhanced Terrain Layer and Terrain Ruler are installed as well. As neither of those modules has been ported to v11, it's unlikely that the change you mentioned will have any effect nowadays. (Unless you're using an older version of Foundry).

When the coloring breaks, take a look at the JavaScript console (it opens with F12). Are there any errors in there?

**Comment by Lockbox313** [comment]
Nothing in the console.
![image](https://github.com/manuelVo/foundryvtt-drag-ruler/assets/8994813/088bfee3-1571-4ce0-a0f7-da1554dae50f)

**Comment by manuelVo** [comment]
I notice that the ruler doesn't start form the token's center, which is unusual behavior for Drag Ruler. Did you create that ruler by dragging the token or by using Ctrl+Drag?

**Comment by Lockbox313** [comment]
CTRL-Drag is a different way of placing waypoints when drag ruler is used?

**Comment by Lockbox313** [comment]
~~CTRL-Drag is there a different way of placing waypoints when drag ruler is used?~~
Ah-ha apparently so just  I changed the right click action to create waypoint and can now place way points.

**Comment by manuelVo** [comment]
I suppose this is solved then?

**Comment by Lockbox313** [comment]
Yes. It could be helpful to put a note in the README that standard Foundry ctrl-click waypoints don't work with drag ruler.   Thanks for the help.
